modulefiles/srw_common.lua

```diff
@@ -4,19 +4,19 @@ load("libpng/1.6.37")
 
 load("netcdf-c/4.9.2")
 load("netcdf-fortran/4.6.1")
-load("parallelio/2.5.10")
+load("parallelio/2.6.2")
```

Comment:
@natalie-perlin @RatkoVasic-NOAA are these library version changes specific for the macos stack built to enable the srw community test? are these lib updates consistent with the stack used by all other machines in srw_common?

Reply:
This wouldn't work on develop branch (other machines) since SRW/WM are still using spack-stack@1.6.0.

Reply:
i think srw common should not be changed then and explicit lib loads need to go into the mac os modulefile

Reply:
The versions of packages built in spack-stack could be tailored to whatever is needed for the repository. 
This work on MacOS started when there were plans to continue with spack-stack-1.8.0 for the UFS-WM and updated package versions. Going back to use lower versions of the packages is always an option.